indico/modules/events/templates/display/indico/meeting.html

```diff
@@ -61,8 +61,17 @@ <h1 itemprop="name">
         {% endif %}
 
         {% if event.has_note %}
-            {{ render_note(event.note, hidden=not theme_settings.show_notes, can_edit=event.can_manage(session.user),
-                           for_event=event) }}
+            {# Check if url contains ?note=note_id and if so, expand the note #}
+            {% set req = request.args.get('note') %}
+            {% set note_id = event.note.id|string %}
+
+            {% if req == note_id %}
+                {{ render_note(event.note, hidden=false, can_edit=event.can_manage(session.user),
+                               for_event=event) }}
+            {% else %}
+                {{ render_note(event.note, hidden=not theme_settings.show_notes, can_edit=event.can_manage(session.user),
+                               for_event=event) }}
+            {% endif %}
```

Comment:
The only difference from `render_notes()` is that it uses `item.can_edit_note(session.user)` instead of `event.can_manage(session.user)`. Can we unify that somehow? Perhaps it is possible to call `event.can_edit_note(session.user)`?

Reply:
yes, it is possible :)